app/views/book_searches/_book_search_bar.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= form_tag(book_search_path, method: :get, id: 'search_form') do %>
```

Comment:
@paulcsmith, As a result of using a `book_searches` controller does this line change at all. I assumed I could now use `simple_form_for` and remove `method: :get` but if I do both of those things, the functionality breaks.

Reply:
You can probably use simple_form_for but you need method: :get so the form knows it needs to submit as a get still